rustfmt-core/rustfmt-lib/src/format_report_formatter.rs

```diff
@@ -1,6 +1,8 @@
 use crate::config::FileName;
-use crate::formatting::FormattingError;
-use crate::{ErrorKind, FormatReport};
+use crate::{
+    formatting::report::FormatReport,
+    result::{ErrorKind, FormatError},
+};
 use annotate_snippets::display_list::DisplayList;
```

Comment:
Seems there's a decent amount of changes with the latest version of `annotate_snippets` and #4161 was opened to apply those changes to rustfmt. Not sure what the best merge sequencing would be between these two PRs